index.html

```diff
@@ -0,0 +1,143 @@
+          <li><a href="#3" class="listAnchor">Join</a></li>
+          <li><a href="#4" class="listAnchor">Sponsor</a></li>
+          <li><a href="#5" class="listAnchor">News</a></li>
+          <li><a href="#6" class="listAnchor">Go Green</a></li>
+      </ul>
+      <img src="img/buttons/mobile_menu.png" id="btnImg" alt="Hamburger type button to display menu">
+  </nav>
+  <div class='txtContainer'>
+      <h1 class="txtGreeting">"GO Green!"</h1>
+      <div>
+          <h2 class='textTitle'>Africa Green Energy Summit 2022</h2>
+      </div>
+      <div class="txtParagraph">
+          <p>Welcome to our annual Africa Green  Energy summit , The event for this year will be held in Nairobi Kenya.
+            Let us come together and discuss sustainable measures to take towards use of Green energy.All are welcomed..
+          </p>
+      </div>
+      <div class="dateContainer">
+          <h2 class="txtDate">19/10/2022 (Monday) ~ 20/10/2022(Tuesday)</h2>
+      </div>
+      <div class="txtDirection ">
+          <p>@PoliedroCaracas, Caracas - Venezuela.</p>
+      </div>
+  </div>
+</header>
+<section id="mainProgram">
+  <h2 class="mainTitle">Main Program</h2>
+  <div id="programs"></div>
+  <div class="btnContainer">
+      <button type="submit" class="mobileBtn" id="programBtn">Join Go Green Africa initiative Program <br>
+          2022</button>
+  </div>
+  <h3 id="seeProgram" class="hide"><a href="#">SEE THE WHOLE PROGRAMM</a></h3>
+</section>
+<section id="speakers">
+  <div class="speakerTitleContainer">
+      <h2 class="speakersTitle">Featured Speakers</h2>
+  </div>
+  <div id="speakerResume">
+      <div id="exponentSample">
+          <div class="expositor" id="speaker1">
+              <div class="expositorImg"><img src="img/speakers/speaker_01.png" alt="expositor image"></div>
+              <div class="expositorTxt">
+                  <h2 class="expositorH2">Yochai Benkler</h2>
+                  <h3 class="expositorH3">Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School
+                  </h3>
+                  <p class="expositorP">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+              </div>
+          </div>
+          <div class="expositor" id="speaker3">
+              <div class="expositorImg"><img src="img/speakers/speaker_03.png" alt="expositor image"></div>
+              <div class="expositorTxt">
+                  <h2 class="expositorH2">SohYeong Noh</h2>
+                  <h3 class="expositorH3">Director of Art Centre Nabi and a board member of CC Korea</h3>
+                  <p class="expositorP">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+              </div>
+          </div>
+      </div>
+      <div id="exponents" class="hideExponents"></div>
+  </div>
+  <button type="submit" id="seeMore" class="seeMore">MORE <img src="img/buttons/arrow_down.png"
+          alt="Down pointer arrow"></button>
+</section>
```

Comment:
- [ ] I noticed that not all your speakers are added dynamically. Please as demanded by the project requirements, add all speakers dynamically using Javascript.